mlibrary/search
# Adding `Noto Sans` as the default sans-serif `font-family`.

Replacing `Source Sans Pro` with `Noto Sans`
- Also includes replacing `Open Sans`
- Updating `Muli` to `Mulish`

# Overview
To provide better support for non-Latin-based languages, [Noto Sans](https://fonts.google.com/noto/specimen/Noto+Sans) has been added as a replacement to [Source Sans Pro](https://fonts.google.com/specimen/Source+Sans+Pro). This pull request is purely for testing actual data. Refining sizes have not been applied.

## Anything else?
Any instances of [Open Sans](https://fonts.google.com/specimen/Open+Sans) has been replaced by Noto Sans.

## Testing
- Run the tests to make sure they pass (`npm run test`).
  - Break the new/updated unit tests to make sure they're working properly.
- Make sure the PR is consistent in these browsers:
- Check the site and see if the new typeface has been applied.
- Check the site and see how non-Latin-based languages compare to the [live](https://search.lib.umich.edu/everything) site.
